app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] Please ensure that you have a new action in your comment controller, which is to display a form creating a comment that should be created in `/views/comments/new.html.erb` for creating a new instance of a model. ( In other words, when a user wants to create a new comment in the database.)